test/index.js

```diff
@@ -127,9 +124,9 @@ describe('Basic', function() {
       });
     });
 
-    describe('ShadowDOM v0', function() {
```

Comment:
To be more precise: this case is currently not supported by this PR:
```html
<div inert>
   <input>
</div>
```
^ the input will still be reachable via TAB